apps/webapp/src/state/ibc.ts

```diff
@@ -57,32 +62,37 @@ export const createIbcSendSlice = (): SliceCreator<IbcSendSlice> => (set, get) =
         state.ibc.destinationChainAddress = addr;
       });
     },
-    sendIbcWithdraw: async toastFn => {
+    sendIbcWithdraw: async () => {
       set(state => {
         state.send.txInProgress = true;
       });
 
-      const { dismiss, update } = toastFn(buildingTxToast());
+      const txToastId = toast(...buildingTxToast());
 
       try {
         const transactionPlan = await plan(await getPlanRequest(get().ibc));
-        const transaction = await authWitnessBuild(update, { transactionPlan });
-        const { txHash, detectionHeight } = await broadcast(update, { transaction });
+        const transaction = await authWitnessBuild({ transactionPlan }, status =>
+          toast(...buildingTxToast(status, txToastId)),
```

Comment:
Instead of the consumer being required to install the npm package + spread the results of `buildingTxToast()` into it, could we change the api of `buildingTxToast()` to do this for us?

```diff
- toast(...buildingTxToast(status, txToastId))
+ buildingTxToast(status, txToastId)
```